LanPartyHub/GameConfig.xaml.cs

```diff
@@ -86,8 +89,20 @@ private void GameConfigAddSetting(object sender, RoutedEventArgs e)
                 var setting = new KeyValue();
                 setting.Key = key;
                 setting.Value = value;
-                _game.GameSettings.Add(setting);
-                GameManager.SaveSettings();
+
+                Boolean found = false;
```

Comment:
Use Linq for this stuff, it's a lot more readable. What are you trying to do?

Reply:
If the setting being added already is "found" in the GameSettings, the KeyValue is updated instead of Added.

Reply:
Use a linq statement, like var setting = _game.GameSettings.Select(item => item.Key == setting.Key)

if(setting != null)
{
     ... update
} 
else ... it's new